### PR TITLE
Minor things found whilst fiddling

### PR DESCRIPTION
**Pull request by mandoonandy**
* Avoid crashing daemon when VXLAN already exists. This should occur until Issue #609 is fixed.
* Minor instance property methods
* Start moving lock names to one location
* Avoid client errors return floating_gateway in net. This will return None initially since it has not been assigned but
subsequent requests give correct data. PR will be raised to request floating_gateway be assigned on instantiation, rather than network creation on network node.

### CODE REVIEW

File: shakenfist/virt.py [hunk 1]
```diff
@@ -290,9 +291,11 @@ def block_devices(self):
         return self._db_get_attribute('block_devices')
 
     # Implementation
+    def get_lock(self, name, op):
```
Comment (mikalstill) [inline shakenfist/virt.py:294]: Isn't this confusing? Its locking an attribute not the entire object?

**Review by mikalstill — COMMENTED** [review]
One small question, and I've re-run CI which I suspect will pass now that ansible galaxy works again...

File: shakenfist/virt.py (hunk 1 above)
Comment (mandoonandy) [reply shakenfist/virt.py:294]: True. Let me modify that.

**Review by mikalstill — APPROVED** [review]
LGTM

File: shakenfist/virt.py [hunk 2]
```diff
@@ -290,9 +291,11 @@ def block_devices(self):
         return self._db_get_attribute('block_devices')
 
     # Implementation
+    def get_lock_attr(self, name, op):
```
Comment (mikalstill) [inline shakenfist/virt.py:294]: Sorry, brain fart. Shouldn't this be in baseobject.DatabaseBackedObject?

**Review by mikalstill — COMMENTED** [review]
Sorry, thought of something.

File: shakenfist/virt.py (hunk 2 above)
Comment (mandoonandy) [reply shakenfist/virt.py]: Good point. The PR auto closed whilst I was messing with my personal develop branch. Will reopen.

**Pull request closed without merging**